<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data</class>
 <widget class="QFrame" name="Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="14">
    <widget class="QPushButton" name="DColor10">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="SelectedFile">
     <property name="minimumSize">
      <size>
       <width>260</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>260</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 255, 240);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="DColor01">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="DataType">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 255, 240);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="DataShow">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>  Type : </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="13">
    <widget class="QPushButton" name="DColor09">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QPushButton" name="DColor06">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <widget class="QPushButton" name="DColor08">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="QPushButton" name="DColor07">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QPushButton" name="DColor05">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="DColor04">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="DColor03">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="DColor02">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="FileSelect">
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item row="0" column="22">
    <widget class="QPushButton" name="DColor18">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="20">
    <widget class="QPushButton" name="DColor16">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="19">
    <widget class="QPushButton" name="DColor15">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="23">
    <widget class="QPushButton" name="DColor19">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="24">
    <widget class="QPushButton" name="DColor20">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="18">
    <widget class="QPushButton" name="DColor14">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="21">
    <widget class="QPushButton" name="DColor17">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="17">
    <widget class="QPushButton" name="DColor13">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="16">
    <widget class="QPushButton" name="DColor12">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="15">
    <widget class="QPushButton" name="DColor11">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="34">
    <widget class="QPushButton" name="DColor30">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="33">
    <widget class="QPushButton" name="DColor29">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="32">
    <widget class="QPushButton" name="DColor28">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="31">
    <widget class="QPushButton" name="DColor27">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="30">
    <widget class="QPushButton" name="DColor26">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="29">
    <widget class="QPushButton" name="DColor25">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="28">
    <widget class="QPushButton" name="DColor24">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="27">
    <widget class="QPushButton" name="DColor23">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="26">
    <widget class="QPushButton" name="DColor22">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="25">
    <widget class="QPushButton" name="DColor21">
     <property name="minimumSize">
      <size>
       <width>6</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>FileSelect</tabstop>
  <tabstop>SelectedFile</tabstop>
  <tabstop>DataShow</tabstop>
  <tabstop>DColor01</tabstop>
  <tabstop>DColor09</tabstop>
  <tabstop>DColor06</tabstop>
  <tabstop>DColor08</tabstop>
  <tabstop>DColor07</tabstop>
  <tabstop>DColor05</tabstop>
  <tabstop>DColor04</tabstop>
  <tabstop>DColor03</tabstop>
  <tabstop>DColor02</tabstop>
  <tabstop>DColor10</tabstop>
  <tabstop>DColor18</tabstop>
  <tabstop>DColor16</tabstop>
  <tabstop>DColor15</tabstop>
  <tabstop>DColor19</tabstop>
  <tabstop>DColor20</tabstop>
  <tabstop>DColor14</tabstop>
  <tabstop>DColor17</tabstop>
  <tabstop>DColor13</tabstop>
  <tabstop>DColor12</tabstop>
  <tabstop>DColor11</tabstop>
  <tabstop>DColor30</tabstop>
  <tabstop>DColor29</tabstop>
  <tabstop>DColor28</tabstop>
  <tabstop>DColor27</tabstop>
  <tabstop>DColor26</tabstop>
  <tabstop>DColor25</tabstop>
  <tabstop>DColor24</tabstop>
  <tabstop>DColor23</tabstop>
  <tabstop>DColor22</tabstop>
  <tabstop>DColor21</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
